packages/redux/src/products/types/actions/productOutfits.types.ts

```diff
@@ -0,0 +1,35 @@
+import type * as actionTypes from '../../actionTypes';
+import type { Action } from 'redux';
+import type { BlackoutError } from '@farfetch/blackout-client';
+import type { NormalizedSchema } from 'normalizr';
+import type { ProductEntity } from '../../../entities/types';
+
+export interface FetchProductOutfitsRequestAction extends Action {
```

Comment:
These types are not being used in fetchProductOutfitsFactory when dispatching. Can you add them? You can take a look at `fetchAddressPredictionDetailsFactory` implementation for an example.

Reply:
You are right. Added this to the `fetchProductOutfitsFactory`
Thank you